Scripts/AzPowerShellScript/CreateVM.ps1

```diff
@@ -2,57 +2,59 @@
 [CmdletBinding(DefaultParametersetName='None')]
 Param(
 	[Parameter(Mandatory=$True)]
-    [String]$VNetAddressSpace,
-	[String]$NicPrivateIp,
+	[String]$ResourceGroupName,
+	[Parameter(Mandatory=$True)]
+	[String]$VmName,
+	[Parameter(Mandatory=$True)]
+	[String]$VMUserName,
+	[Parameter(Mandatory=$True)]
+	[String]$VMPassword,
+	[Parameter(Mandatory=$True)]
+	[ValidateSet("Windows", "Linux")]
+	[String]$OS,
 	[Parameter(Mandatory=$True)]
 	[String]$VHDPath,
```

Comment:
We do not need 2 parameters to get the vhd full path.

Reply:
the vhd file name is required for creating a managed disk (not including path)
vhd file name and vhd file path is required for azopy